kola/tests/docker/torcx_manifest_pkgs.go

```diff
@@ -20,7 +20,7 @@ import (
 	"strings"
 
 	"github.com/coreos/go-semver/semver"
-	ignition "github.com/coreos/ignition/config/types"
+	ignition "github.com/coreos/ignition/config/v2_1/types"
```

Comment:
Rather than naming the package `ignition`, should we alias it to `v21types` like we do in `platform/conf`?

Reply:
There's not room for confusion here, and I think `ignition` reads clearer for what it's used as here.

In `conf` there's multiple `types` imported so it helps to clarify that confusion, but I don't see that problem here.